WTWSMS/history/characters/roman.txt

```diff
@@ -1985,17 +1985,13 @@
 	420.1.1 = {
 		birth="420.1.1"
 	}
-	492.1.1 = {
-	name="Gelasius"
-	# AKA: Acacius
-	}
```

Comment:
What happened here?

Reply:
I removed it on purpose. Was his birth name really Acacius?

Reply:
If that is was we have, presumably yes.

Reply:
I don't think it's an effect of deep research. Gelasius didn't change his name when he became Pope, his name was Gelasius... Maybe someone used Acacius character as a base?

Reply:
These Popes are outside of CK2 timeline, so PDX didn't really care much about them.

Reply:
If I'm not wrong, the mod adds some Popes too.

Reply:
This Pope is from vanilla, I checked